coderd/database/queries/workspaceappstats.sql

```diff
@@ -0,0 +1,24 @@
+-- name: InsertWorkspaceAppStats :exec
+INSERT INTO
```

Comment:
Can you scope this PR for collecting `app_stats` and then focus on the Insights endpoint in a follow-up? Otherwise, this PR will become too large and hard to review. As an extra benefit, we will start collecting stats earlier :)

Reply:
Yup, that's the plan 👍